commands/utils/consts.go

```diff
@@ -57,6 +57,7 @@ const (
 	// Environment variables to control Frogbot's functionality.
 	IncludeAllVulnerabilitiesEnv = "JF_INCLUDE_ALL_VULNERABILITIES"
 	FailOnSecurityIssuesEnv      = "JF_FAIL"
+	UseWrapperEnv                = "JF_USE_WRAPPER"
```

Comment:
UseWrapperEnv --> UseGradleWrapperEnv
JF_USE_WRAPPER --> JF_USE_GRADLE_WRAPPER

Let's also change the template accordingly.